cypress/e2e/8-motions.js

```diff
@@ -178,6 +175,65 @@ describe('User can create motions via UAC', () => {
     });
   });
 
+  it('Claiming Stakes', () => {
+    cy.login();
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
```

Comment:
couldn't this be destructured more cleanly at the start of the file ?

Ie:
```js
const { colony: { name: colonyName }} = Cypress.config();

// ... then it can be used like
cy.visit(`/colony/${colonyName}`);
```

It makes the code a tad more readable

Reply:
Nice and clean!